macrosynergy/pnl/notional_positions.py

```diff
@@ -111,7 +111,7 @@ def notional_positions(
     dollar_per_signal: Numeric = 1.0,
     leverage: Optional[Numeric] = None,
     vol_target: Optional[Numeric] = None,
-    rebal_freq: str = "m",
+    rebal_freq: str = "M",
```

Comment:
Why capital "M" - standard JPMaQS/Macrosynergy notation would be lower case `m` - we need to make it consistent then across the Macrosynergy package.

Reply:
We use capitals because pandas uses it. `macrosynergy.management.utils.core._map_to_business_day_frequency` exists to provide that mapping consistently across package functions (those that have been refactored to do so anyway).